Documentation/Troubleshooting/ceph-common-issues.md

```diff
@@ -507,6 +507,75 @@ You don't need to restart the pod, the effect will be immediate.
 
 To disable the logging on file, simply set `log_to_file` to `false`.
 
+## Collect perf data of a ceph process at runtime
+
```

Comment:
I think it might be worth adding a warning here that this is an advanced section
```
!!! warn
    This is an advanced topic please be aware of what you are doing...
```
(https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)

@travisn @gauravsitlani what are your thoughts?

Reply:
@galexrt agreed, expert guidance should be referred or the users should be aware what they're doing here.

Reply:
Actually, how about if we make this a separate page? It really isn't a common troubleshooting page. Perhaps it could be titled "Performance Profiling".

Reply:
@travisn sure sounds good. I'll make the changes